Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -514,6 +516,39 @@ class BrowserViewController: UIViewController {
         }
     }
 
+    private func animateBookmarkStar() {
+        let duration = 0.5
```

Comment:
I would move this up into the BrowserViewControllerUX struct just so we can keep all of the style properties in one place